Dockerfile

```
@@ -74,7 +74,9 @@ RUN \
     rm    storage/framework/views/* 2> /dev/null || true && \
     rm    storage/logs/* 2> /dev/null || true && \
     chown -R www-data:www-data /var/www/html/Lychee && \
-    chmod -R g+ws storage/image-jobs storage/livewire-tmp && \
+    chmod -R g+ws storage/image-jobs || true && \
+    chmod -R g+ws storage/livewire-tmp || true && \
+    chmod -R g+ws storage/lychee-tmp && \

```

Do we need this for v5 as it won't exist there?
```suggestion
    chmod -R g+ws storage/lychee-tmp || true&& \

```

---

Indeed. Forgot.

---

That is most likely why it is crashing